docs/en/guides/pipeline-integrations/administration/setting-python-3-desktop.md

```diff
@@ -42,15 +44,15 @@ lang: en
 
```

Comment:
Line 40 it should mention Python 2
- Restart the {% include product %} Desktop application. Now, you should see that the Python version has been updated to run Python 3.